app/pb_kits/playbook/pb_date_picker/_date_picker.scss

```diff
@@ -0,0 +1,1018 @@
+// @import "~flatpickr/dist/flatpickr.css";
+@import "../pb_textarea/textarea_mixin";
+@import "../pb_title/title_mixin";
+@import "../tokens/colors";
+@import "../pb_caption/caption_mixin";
+
+[class^=pb_date_picker_kit] {
+  // input styles
+  [class^=pb_caption_kit] {
+    margin-bottom: $space_xs;
+    display: block;
+  }
+  .input_wrapper {
+    margin-bottom: $space_sm;
+    position: relative;
+    
+    .flatpickr-wrapper {
+      display: block;
+      input {
+        text-overflow: ellipsis;
+        padding-right: 45px !important;
+      }
+      input::placeholder {
+        @include pb_body_light;
+      }
+      
+  
+      > input:first-child {
+        @include pb_textarea_light;
+        @include pb_title_4;
+        overflow: hidden;
+      }
+  
+      input:focus, input:-webkit-autofill:focus {
+        @include pb_textarea_focus;
+        -webkit-box-shadow: 0 0 0px 1000px $focus_input_light inset;
+        transition: background-color 5000s ease-in-out 0s;
+      }
+    }
+    // Icon for select tags
+    .month-dropdown-icon {
+      align-self: center;
+      margin-top: 5px;
+      width: 10px;
+      position: relative;
+      right: 10px;
+      pointer-events: none;
+    }
+    .year-dropdown-icon {
+      align-self: center;
+      margin-top: 5px;
+      width: 10px;
+      position: relative;
+      right: 12px;
+      pointer-events: none;
+    }
+    // calendar icon styles
+    .cal_icon_wrapper {
+      position: absolute;
+      top: 1px;
+      right: 0px;
+      border-left: 1px solid $border_light;
+      width: 45px;
+      height: 43.2px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding-left: 11px;
+      color: $text_lt_light;
+      &:hover {
+        cursor: pointer;
+      }
+    }
+    .cal_icon {
+      width: 20px;
+    }
+
+    // Header Styles
+    $header_height: 40px;
+
+    .flatpickr-months {
+      // padding: 5px;
+      // padding-bottom: 5px;
+      border-bottom: solid 1px $border_light;
+    }
+    .flatpickr-month {
+      height: $header_height;
+    }
+    .flatpickr-current-month {
+      left: 10px;
+      display: flex;
+      // justify-content: flex-start;
+      padding-top: 0px;
+    }
+    .flatpickr-monthDropdown-months {
+      appearance: none;
+      padding-left: 0px;
+      padding-right: 14px;
+      margin-left: 15px;
+      height: $header_height;
```

Comment:
This top area is pretty strange. @kre8sions can the selects be improved with flexbox or grid?

Reply:
Agreed top area is pretty strange.  Flatpickr's div/tag structure is messy.  The wrapper for the selects is already `display: flex` but the month prev and month next arrows are `position: absolute`